src/main.qml

```diff
@@ -52,9 +53,7 @@ Application {
     property int pulseWidthArc: pulseToggle ?
                                  0 :
                                  -6
-    property int pulseWidth: pulseToggle ?
-                                 app.height * 0.20 :
-                                 app.height * 0.26
+    property int pulseWidth: pulseToggle ? Dims.h(20) : Dims.h(26)
```

Comment:
Need to nit pick. 
As seen in above lines, i gave new lines for the then/else decisions o make it easier for me and other beginners to spot the `?`.
Personally, i have no strong opinion (anymore) over short one liner ifs or the then/else seperated in new lines. I can spot them by now. But maybe other beginners might find new lines helpful.
But now its inconsistent use. Please either change to new lines here or adapt the other short ifs into one liners imho.

Reply:
That's also fine :)